docs/platforms/react-native/configuration/options.mdx

```diff
@@ -259,6 +259,18 @@ A function responsible for determining the percentage chance a given transaction
 
 </ConfigKey>
 
+<ConfigKey name="trace-propagation-targets">
+
+An optional property that controls which downstream services receive tracing data, in the form of a `sentry-trace` and a `baggage` header attached to any outgoing HTTP requests.
+
+The option may contain a list of strings or regex against which the URLs of outgoing requests are matched.
+If one of the entries in the list matches the URL of an outgoing request, trace data will be attached to that request.
+String entries do not have to be full matches, meaning the URL of a request is matched when it _contains_ a string provided through the option.
+
+If <PlatformIdentifier name="trace-propagation-targets" /> is not provided, trace data is attached to every outgoing request from the instrumented client.
```

Comment:
RN behaves like browser (only same origin, which means only localhost).

This is a bug, but since would be breaking change we should include it in next major.

Till then I would remove/adjust the last sentence.

Reply:
updated to https://github.com/getsentry/sentry-docs/pull/10305/files#diff-e93f72184e1d71df00287acd133e707209b032ce102a6317b22495543b4b5694R270 - wdyt?

Reply:
Yes, that looks good.